app/views/home/_signed_in_jobseeker.html.slim

```diff
@@ -2,4 +2,7 @@ h2.govuk-heading-m = t(".title")
 
 p = t(".description")
 
-= govuk_link_to t("buttons.view_account"), jobseekers_saved_jobs_path, class: "govuk-button--secondary govuk-!-margin-bottom-0", button: true
+- if current_variant?(:"2012_02_jobseeker_account_cta_test", :colour)
+  = govuk_link_to t("buttons.view_account"), jobseekers_saved_jobs_path, class: "govuk-button--primary govuk-!-margin-bottom-0", button: true
```

Comment:
What's the purpose of `govuk-button--primary`?

Reply:
If we use the secondary button, the colour of the button will be grey (against a grey background colour)

Reply:
My point is that `govuk-button--primary` doesn't do anything (you can just remove `govuk-button--secondary`)